src/scripts/temporaryBattle/TemporaryBattle.ts

```diff
@@ -66,4 +66,8 @@ class TemporaryBattle extends TownContent {
         }
         this.completeRequirements = completeRequirements;
     }
+
+    public getPokemonList() {
```

Comment:
Same reasoning as with gym pokemons
```suggestion
    get pokemons(): GymPokemon[] {
```